FirebaseAuth/Tests/Sample/AuthSample.xcodeproj/project.pbxproj

```diff
@@ -214,13 +216,6 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
-		A00A0EC6CCFF06F1AC46A4F9 /* Pods */ = {
```

Comment:
These get left in file if `pod deprecate` is run when Xcode has the xcworkspace open.